docs/contributing/events/security_alerts/symbols_collision.md

```diff
@@ -0,0 +1,52 @@
+# symbols_collision
+
+## Intro
+symbols_collision - a shared object loaded to a process has collisions in exported symbols with another loaded shared object.
+
+## Description
+An event marking that a shared object loaded to current process, and have collisions of exported symbols
+with other shared object already loaded to the process. This event can help in inform on the 
+occasion that a shared object tries to override some symbol of another library.
+
+### Configuring the event
+The event is configured using arguments filtering.
+#### symbols
+Configure the watched symbols that upon collision will trigger the event.
+Specify the full name of the symbol for each symbol.
+Notice that only watched symbols will be outputed by the event, and the default is watching all symbols.
+The use is only with the `=` or `!=` operators, and wildcards aren't supported.
+
+## Arguments
+* `loaded_path`:`const char*`[K] - the path of the file loaded.
+* `collision_path`:`const char*`[K,TOCTOU] - the path of the file already loaded, which has collision with the new loaded one.
+* `symbols`:`const char*const*`[U,TOCTOU] - list of symbols collided between the files.
+
+## Dependency Events
+### shared_object_loaded
+The event of shared object loading triggers this event, and supplies the information on the
+shared object necessary to examine its shared objects.
+
+### sched_process_exec
+Used by tracee to maintain mount NS cache, used in this event to get to processes file system.
+Also, used to maintain the cache used by the event for performance improvment.
+
+## Example Use Case
```

Comment:
i think a cmdline example of how to use the event with the filters is very important here